app/src/main/java/io/github/droidkaigi/confsched/prefs/DefaultPrefsSchema.java

```diff
@@ -5,12 +5,12 @@
 
 @Table(name = "io.github.droidkaigi.confsched_preferences")
 public abstract class DefaultPrefsSchema {
```

Comment:
Can we change it to interface? If we can do that we don't have to make each variables final.